README.md

```diff
@@ -69,3 +69,10 @@ vendor/bin/drush --yes cache:rebuild
 composer check-coding-standards
 composer apply-coding-standards
 ```
+
+
+## Build assets
+
+```sh
+docker run -v ${PWD}:/app itkdev/yarn:latest encore dev
```

Comment:
Maybe we tell that 
```
docker run --volume ${PWD}:/app --workdir /app node:latest yarn install
```
should be run as well. And maybe add some helper scripts to `package.json` (cf. https://github.com/itk-dev/hoeringsportal/blob/develop/web/themes/custom/hoeringsportal/package.json#L16-L17).